.github/workflows/ci.yml

```diff
@@ -217,11 +231,37 @@ jobs:
       - run: sudo chmod 0666 /dev/kvm
 
       - name: Test
-        run: gotestsum --raw-command --ignore-non-json-output-lines --junitfile junit.xml -- vimto -kernel :${{ matrix.tag }} -- go test -short -count 1 -json ./...
-
+        run: gotestsum --raw-command --ignore-non-json-output-lines --junitfile junit.xml -- vimto -kernel :${{ matrix.tag }} -- go test -short -count 1 -json ./... -coverprofile=coverage-${{ matrix.tag }}.out
```

Comment:
Where are these cover profiles uploaded, and how are they merged in to the final result?

Reply:
Sorry, I forgot to add dir for upload. Made changes.